app/src/main/res/layout/recyclerview_item_hf.xml

```diff
@@ -4,16 +4,17 @@
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Why are changes made in this file?

Reply:
Matt is working on this file. I was told to leave all the changes on recycler_view_hf as is, and Matt can resolve the conflict later.